Produlytics/ui/back-end/src/main/java/it/deltax/produlytics/uibackend/admins/devices/business/services/InsertCharacteristicService.java

```diff
@@ -54,11 +54,11 @@ public int insertByDevice(int deviceId, NewCharacteristic characteristic)
       throw new BusinessException("duplicateCharacteristicName", ErrorType.GENERIC);
     }
 
-    if (characteristic.autoAdjust() && characteristic.sampleSize().isEmpty()) {
-      characteristic = characteristic.toBuilder()
-          .withSampleSize(OptionalInt.of(0))
-          .build();
-    }
+//    if (characteristic.autoAdjust() && characteristic.sampleSize().isEmpty()) {
+//      characteristic = characteristic.toBuilder()
+//          .withSampleSize(OptionalInt.of(0))
+//          .build();
+//    }
```

Comment:
Beh cancelliamo, non commentiamo codice. In caso lo andiamo a ripescare in commit vecchi.

Reply:
Li avevo commentati mentre stavo testando e poi mi sono dimenticato di cancellarli. Ora tolgo